packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -324,6 +324,20 @@ mixin WidgetsBinding on BindingBase, ServicesBinding, SchedulerBinding, GestureB
         },
       );
 
+      // Register the ability to quickly mark elements as dirty.
+      registerServiceExtension(
+        name: 'fastReassemble',
+        callback: (Map<String, Object> params) async {
+          final String className = params['class'];
+          for (Element element in Element.debugTrackedElements) {
+            if (element.widget.runtimeType.toString() == className) {
```

Comment:
How does this work with generics? (though not sure how popular generic widgets are)

Reply:
Good point, maybe contains in would a better approach

Reply:
how about `element.widget.runtimeType.startsWith(className)`
That should be sufficient.